features/all-positions/AllLiquidations.tsx

```diff
@@ -172,19 +172,19 @@ const AllLiquidations = () => {
     // then sorts the positions according to selected sort column,
     // and finally slices the array based on pagination selection.
     const reformattedLiquidationData = Object.keys(liquidations)
-      .filter((sponsor: string) => {
+      .filter((sponsorAddressPlusId: string) => {
         return (
-          liquidations[sponsor]?.maxDisputablePrice &&
-          liquidations[sponsor]?.tokensLiquidated &&
-          liquidations[sponsor]?.lockedCollateral &&
-          liquidations[sponsor]?.liquidationTimestamp
+          liquidations[sponsorAddressPlusId]?.maxDisputablePrice &&
+          liquidations[sponsorAddressPlusId]?.tokensLiquidated &&
+          liquidations[sponsorAddressPlusId]?.lockedCollateral &&
+          liquidations[sponsorAddressPlusId]?.liquidationTimestamp
```

Comment:
Thanks, this is easier to understand.

nit: just as a shortener, maybe we could do `sponsorPlusId` as you did above?